src/server/resources/queries/vehicle/VEHICLE_NO_MOT.sql

```diff
@@ -0,0 +1,20 @@
+select veh.registration
+from vehicle as veh
+left join mot_test_current as mtc on veh.id = mtc.vehicle_id
+and veh.registration not like "%-%" -- exclude dodgy test data on ACPT
```

Comment:
The first and after the left join should have been a WHERE.

This script looks flawed.   At the time it was written there would have been few [DVSA] vehicles in existence that did not have a test associated with them - I'd expect the majority of these would have been vehicles that TSS moved tests away from due to tester errors.

Since them DQ team did create a bunch of vehicles (or at least planned to create vehicles) before they were tested so these may actually exist presently.

If the intention of this is to find vehicles with no valid MOT, then I expect a different query is required here.